sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
> I'm afraid I had assumed the opposite as I was asked to change from multiple-of-four indent to hanging indent in this PR (see the outdated diff above).

I see. I guess @leosartaj is on a different boat with that one.

 > There is too much variety in coding styles (including hanging 
indents) even in Sympy.

That's true.

> Since this PR is already merged,

Yeah, I noticed too late.

 > you can make the changes in another PR if you want.

That would come as a big style clean-up, not in a piecemeal fashion :-)
The important point is that people know so the style is somewhat more 
consistent over time.